field-testing/src/lib.rs

```diff
@@ -91,6 +91,27 @@ pub fn test_two_adic_coset_zerofier<F: TwoAdicField>() {
     }
 }
 
+pub fn test_two_adic_generator_consistency<F: TwoAdicField>() {
+    let log_n = F::TWO_ADICITY;
+    let g = F::two_adic_generator(log_n);
+    for i in 0..log_n {
```

Comment:
nit: can it include `log_n`, i.e. `0..=log_n`?

Reply:
fixed, thanks.